components/Livestream.tsx

```diff
@@ -16,7 +16,7 @@ function Livestream({ currentLeague = 'shl' }: { currentLeague: string }): JSX.E
   const videoID = livestreamData ? livestreamData[0].videoId : '';
 
   if (livestreamData) {
-    if (livestreamData[0].isLive === 'live') {
+    if (livestreamData[0].isLive === 'live' && isLive !== true) {
       setIsLive(true);
     }
   }
```

Comment:
We should actually probably be doing this with a useEffect hook.

so we could change it to 
```js
useEffect(() => {
  if (livestreamData[0].isLive === 'live) {
    setIsLive(true);
  }
}, [livestreamData]);

```

That way it only runs this code when livestreamData has changed and so if we need to change things around later, we can work within this hook. Be sure to include importing useEffect from React above if you decide to change to this

Reply:
works for me.  useEffect() is something else I have not seen before.  Still learning react

Reply:
The change is in.